modular_skyrat/modules/modular_weapons/code/melee.dm

```diff
@@ -9,6 +10,11 @@
 	new /obj/item/melee/sabre/cargo(src)
 	update_appearance()
 
+/obj/item/melee/sabre
+	force = 20
+	wound_bonus = 5 // 5 less
+	bare_wound_bonus = 20 // 5 less: Both down slightly, to make up for the damage buff, since it'd get a bit wacky ontop of the armor pen.
```

Comment:
If we're buffing TG's Captain's sabre, the original definition should probably have a comment of it so nobody comes by it later and wonders why it's behaving abnormally. 
Just to make it easier for you you can copy-paste this: 
`//SKYRAT EDIT - Buffed in modular_skyrat/modules/modular_weapons/code/melee.dm`

Additionally, this specific chunk I marked should _probably_ have original values for reference

Reply:
word